### PR TITLE
INS1012

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
A new repository (recently inventorized).

### CODE REVIEW

File: new/INS1012AddiLaqmaMaryam.xml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                    <settlement when="2024-03-15" ref="LOC7446Alaqma" type="qushat">qušat ʿĀddi Lāq̲mā</settlement>
+                    <settlement when="2024-03-15" ref="LOC7447GolgolDaro" type="area">locality Golgol Dāʿro</settlement>
+                    <location>
+                        <geo>14.378267 39.354153</geo>
+                        <height unit="m">2227.0</height>
+                    </location>
+                    <note>
+                        <date type="foundation" notBefore="1350" notAfter="1449">Possibly around mid-14th century.</date>
+                        <ab type="history">Dabr. According to the local tradition, the first church was built in the 12th century.
+                            Before,  the area is said to have been occupied by the 'people who came from Israel' and by Kalaw-Balaw.
```
Comment (CarstenHoffmannMarburg) [inline new/INS1012AddiLaqmaMaryam.xml:68]: Is Kalaw-Balaw also a people or is it a (legendary) person?

File: new/INS1012AddiLaqmaMaryam.xml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                            The present church is said to have been built some 120 years ago. Some archaeological traces of early settlements
```
Comment (CarstenHoffmannMarburg) [inline new/INS1012AddiLaqmaMaryam.xml:69]: 'earlier' settlements is better. For 'early' settlements, I do not know, what is the standard of comparison?

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: new/INS1012AddiLaqmaMaryam.xml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                            <persName ref="PRS5578Isaac" type="tabot">Isaac</persName>.
+                        </ab>
+                        
+                        <ab type="description">33 manuscripts were inventarized on <date>30 March 2024</date>
+                            by Getnet Desta, Amanuel Abrha, Hailay Atsbha (Department of Archaeology and Heritage Management, 
+                            Adigrat University), and Hagos Gebremariam (Department of Sociology, Adigrat University).</ab>
+                    </note>
+                </place>
+                <listRelation>
+                    <relation name="skos:exactMatch" active="PRS1269Abraham" passive="LOC7446Alaqma"/>
```
Comment (eu-genia) [inline new/INS1012AddiLaqmaMaryam.xml:88]: active="INS1012AddiLaqmaMaryam"

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/INS1012AddiLaqmaMaryam.xml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="church religiousSite" subtype="institution">                    
+                    <placeName xml:id="n1" xml:lang="gez">ዓዲ፡ ላቕማ፡ ቅድስት፡ ማርያም፡</placeName>
+                    <placeName corresp="#n1" xml:lang="en">ʿĀddi Lāq̲mā Qǝddǝst Māryām</placeName>
+                    <placeName xml:id="n2" xml:lang="gez">ዓዲ፡ ላቕማ፡ ማርያም፡ ዘዲበ፡ ኰኵሕ፡</placeName>
+                    <placeName corresp="#n2" xml:lang="en">ʿĀddi Lāq̲mā Māryām Wadiba Kʷakʷǝḥ</placeName>
```
Comment (thea-m) [inline new/INS1012AddiLaqmaMaryam.xml:51]: Zadiba or Wadiba?

File: new/INS1012AddiLaqmaMaryam.xml [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+                    </location>
+                    <note>
+                        <date type="foundation" notBefore="1350" notAfter="1449">Possibly around mid-14th century.</date>
+                        <ab type="history">Dabr. According to the local tradition, the first church was built in the 12th century.
+                            Before,  the area is said to have been occupied by the 'people who came from Israel' and by Kalaw-Balaw.
+                            The present church is said to have been built some 120 years ago. Some archaeological traces of early settlements
+                            were discovered in the area around the church. 
+                        </ab>
+                        <ab type="tabot">
+                            <measure quantity="2">4</measure> four main  tābots,
```
Comment (thea-m) [inline new/INS1012AddiLaqmaMaryam.xml:73]: 2 or 4? If four, it is encoded twice?

File: new/INS1012AddiLaqmaMaryam.xml (hunk 3 above)
Comment (thea-m) [reply new/INS1012AddiLaqmaMaryam.xml:88]: I don't understand the use of skos:exactMatch between a person and a place

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you :)

File: new/INS1012AddiLaqmaMaryam.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/INS1012AddiLaqmaMaryam.xml:88]: changed

File: new/INS1012AddiLaqmaMaryam.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/INS1012AddiLaqmaMaryam.xml:68]: people

**Pull request merged**